gui/src/app/FileEditor/StanFileEditor.tsx

```diff
@@ -107,21 +98,15 @@ const StanFileEditor: FunctionComponent<Props> = ({
     setCompiledUrl,
   ]);
 
-  const [didInitialCompile, setDidInitialCompile] = useState(false);
```

Comment:
It seems like it might be valuable to keep something like this? I think as this is currently implemented, we're doing a fetch every time a file is saved, not just on initial load?

Reply:
No, on each save it does not do a remote fetch unless it found the thing in the cache and needs to verify that it still exists on the server. Reading this small item from localStorage is very inexpensive.

Reply:
Ah, right. Overlooked that